test/e2e/kubernetes/verify.go

```diff
@@ -84,7 +84,7 @@ func (v VerifyNode) Run(ctx context.Context) error {
 	v.Logger.Info("Successfully exec'd nginx -version", "pod", podName)
 
 	v.Logger.Info("Checking logs for nginx output", "pod", podName)
-	logs, err := GetPodLogsWithRetries(ctx, v.K8s, podName, testPodNamespace)
+	logs, err := GetPodLogsWithRetries(ctx, v.K8s, podName, testPodNamespace, &corev1.PodLogOptions{})
```

Comment:
not a blocker
The fact that you had to add the empty string to 3 calls already show that the customization of the logs calls is probably not the common usecase.

In general, I think making interfaces that simplify use for the most common usecase while allowing customization for the rest is a good rule of thumb

In this case, could we make it a variadic argument>